src/main/java/org/sunbird/portal/SbCbExtApplication.java

```diff
@@ -14,7 +14,6 @@
 import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
 
-@EnableJpaRepositories(basePackages = {"org.sunbird"})
```

Comment:
will this affect existing db config loading... 
can you check the other postgress usage and the APIs are working ?